util/hostname.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+//nolint:gochecknoglobals
+var (
+	hostname    string
+	hostnameErr error
+)
+
+const hostnameFile string = "/etc/hostname"
+
+//nolint:gochecknoinits
+func init() {
+	getHostname(hostnameFile)
+}
+
+// Direct replacement for os.Hostname
+func Hostname() (string, error) {
+	return hostname, hostnameErr
+}
+
+// Only return the hostname(may be empty if there was an error)
+func HostnameString() string {
+	return hostname
+}
+
+func getHostname(location string) {
+	hostname, hostnameErr = os.Hostname()
```

Comment:
Do we need the fallback to "/etc/hostname"? Why not just to use the os.Hostname (KISS)?

Reply:
I like to keep it since it provides an simple solution to the previous described problem of consistent hostnames in a multi instance setup.